exercises/concept/wizards-and-warriors/.meta/design.md

```diff
@@ -27,14 +27,22 @@ This exercise's prerequisites Concepts are:
 - `classes`
 - `strings`
 - `if-else-statements`
+- `ternary-operators`
 
 ## Representer
 
 This exercise does not require any specific representation logic to be added to the [representer][representer-java].
 
 ## Analyzer
 
-This exercise does not require any specific analyzer logic to be added to the [analyzer][analyzer-java].
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `actionable`: If the solution uses an `if` statement in the `getDamagePoints` method for Wizard and Warrior classes, instruct the student to use a ternary expression instead.
```

Comment:
Not sure about this one to be honest. It's also pretty off-topic for this concept exercise, so maybe we should leave it out.